### PR TITLE
feat: Improvements in helm chart

### PR DESCRIPTION
**Pull request by rvgud**
## 📑 Description
Improves overall helm chart, adding options to add extra volumes, options to disable pv or pvc, options to add health probes

### CODE REVIEW

**Review by pehlicd — COMMENTED** [review]
please also run `pre-commit run -a` for updating helm doc

**Comment by rvgud** [comment]
pre-commit run -a

File: charts/keep/values.yaml [hunk 1]
```diff
@@ -76,6 +76,19 @@ backend:
   nodeSelector: {}
   tolerations: []
   affinity: {}
+  healthCheck:
```
[the remark this answers is not in the document]
Comment (pehlicd) [reply charts/keep/values.yaml:79]: Ok I got it, then we can leave it in that way, thanks for the contribution

**Review by pehlicd — APPROVED** [review]
LGTM!

**Pull request merged**